catalyst-gateway/bin/Cargo.toml

```diff
@@ -19,7 +19,7 @@ cardano-chain-follower = { version = "0.0.8", git = "https://github.com/input-ou
 rbac-registration = { version = "0.0.4", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250220-00" }
 catalyst-types = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250220-00" }
 cardano-blockchain-types = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250220-00" }
-catalyst-signed-doc = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250220-00" }
+catalyst-signed-doc = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "r20250225-00" }
```

Comment:
Why do we need a new release tag, but the same version number?  Are they the same or different?  If different why isn't the version number bumped?

Reply:
Oh, my mistake, I have not bumped version of the `catalyst-signed-doc` crate. Will do that for the next release